DATA/production/qc-async/trd.json

```
@@ -62,6 +62,35 @@
           "pulseheightpeaklower": "1.0",
           "pulseheightpeakupper": "5.0"
         }
+      },
+      "PulseHeightTrackMatch": {
+        "active": "false",
+        "className": "o2::quality_control_modules::trd::PulseHeightTrackMatch",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS/0;tracklets:TRD/TRACKLETS/0;triggers:TRD/TRKTRGRD/0;tracks:TRD/MATCH_ITSTPC;trigrectrk:TRD/TRGREC_ITSTPC"
+        },
+        "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        },
+        "location": "local",
+        "localMachines": [
+          "epn",
+          "localhost"
+        ],
+        "remoteMachine": "alio2-cr1-qc05.cern.ch",
+        "remotePort": "47745",
+        "mergingMode": "delta",
+        "mergerCycleMultiplier": "2",
+        "localControl": "odc"
```

```suggestion
        }
```
You don't need all of that in async processing

---

as comment above.

---

removed